google-compute-engine/src/test/java/org/jclouds/googlecomputeengine/compute/GoogleComputeEngineServiceLiveTest.java

```diff
@@ -67,6 +74,34 @@ public void testListHardwareProfiles() throws Exception {
       }
    }
 
+   public void testCreateNodeWithSsd() throws Exception {
+      String group = this.group + "ssd";
+      try {
+         TemplateOptions options = client.templateOptions();
+
+         options.as(GoogleComputeEngineTemplateOptions.class).bootDiskType("pd-ssd");
+
+         // create a node
+         Set<? extends NodeMetadata> nodes =
+               client.createNodesInGroup(group, 1, options);
+         assertEquals(nodes.size(), 1, "One node should have been created");
+
+         // Verify the disk on the instance is an ssd.
+         NodeMetadata node = Iterables.get(nodes, 0);
+         GoogleComputeEngineApi api = client.getContext().unwrapApi(GoogleComputeEngineApi.class);
+         Instance instance = api.instancesInZone(node.getLocation().getId()).get(node.getName());
+         Disk disk = api.disksInZone(node.getLocation().getId()).get(toName(instance.disks().get(0).source()));
+         assertTrue(disk.type().toString().endsWith("pd-ssd"));
+
+         // delete the node
+         client.destroyNodesMatching(NodePredicates.inGroup(group));
+
+      } finally {
+         client.destroyNodesMatching(NodePredicates.inGroup(group));
+         if (context != null)
```

Comment:
This context (as opposed to the one used in the invalid auth test) is reused by all tests. Don't close it as its life cycle is managed in the superclasses.